src/include/rbd/librbd.h

```diff
@@ -146,7 +146,7 @@ typedef enum {
 
 typedef struct {
   char *name;
-  int64_t pool;
+  char *pool;
```

Comment:
Why the change to a C-string? I think it's safe to let the caller decode the pool name -- plus, the pool id is actually more helpful if trying to create IoCtx's because it's safer from races.